tests/schedulers/airflow/test_plan.py

```diff
@@ -525,7 +526,7 @@ def test_plan_dag_state(snapshot: Snapshot, sushi_context: Context, random_name)
     environment_name = random_name()
     environment = Environment(
         name=environment_name,
-        snapshots=[snapshot.table_info],
+        snapshots=[json.loads(snapshot.table_info.json())],
```

Comment:
I don't think this is a good test modification. I'd say we should only test that it's a list of dict on deserialization, since it will always be serialized from the `SnapshotTableInfo` instance.